emojisum-rs/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "emojisum"
+version = "0.0.1"
+authors = ["hello@organism.ai"]
+description = "Emoji based cheksums"
```

Comment:
s/cheksums/checksums/

Reply:
> ok. I've sat on this a while. Perhaps it would be cleaner to make this library into its own repo under the emojisum org. That way it could nicely have its own travis-ci.org checks as well. Thoughts?
> Also, can you remove the copy of the emojimap.json?

I think that is fine, we can do that